### PR TITLE
Add warning that lsp doesn't work with Julia 1.0 yet

### PR DESCRIPTION
**Pull request by non-Jedi**
Interested parties can track progress toward Julia 1.0 lsp at https://github.com/non-Jedi/lsp-julia/issues/3.

### CODE REVIEW

File: layers/+lang/julia/README.org [hunk 1]
```diff
@@ -69,6 +69,9 @@ This layer also includes experimental support for =lsp-mode=. To enable language
 server functionality with this layer, follow [[#lsp][the instructions for installation]],
 and then install this layer with:
 
+Warning: [[https://github.com/JuliaEditorSupport/LanguageServer.jl][ =LanguageServer.jl=]] has not yet been released for Julia 1.0. If using
```
Comment (d12frosted) [inline layers/+lang/julia/README.org]: You have an extra space here. But I've fixed that for you.

Comment (d12frosted) [reply layers/+lang/julia/README.org]: Also, I've moved it a little bit down, so it doesn't interfere with enabling instructions.

**Comment by d12frosted** [comment]
Thank you :+1: Cherry-picked into `develop`, you can safely delete your branch.

**Comment by d12frosted** [comment]
P. S. 🎉 congratulations on your first contribution!

**Comment by non-Jedi** [comment]
Well second one anyway, but thanks. :)

https://github.com/syl20bnr/spacemacs/pull/10444

And thanks for cleaning it up for me.

**Pull request closed without merging**